src/Command/UsersBaseCommand.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\User;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Security\Core\Exception\UserNotFoundException;
+
+abstract class UsersBaseCommand extends Command
```

Comment:
```suggestion
abstract class AbstractUsersCommand extends Command
```

See https://www.php-fig.org/bylaws/psr-naming-conventions/